test/cmdline_test.py

```diff
@@ -219,6 +221,19 @@ def test_subclasses_2(self):
         self.assertEquals(task.x, 'xyz')
 
 
+@mock.patch('luigi.cmdline.luigi.interface')
+class LuigiMainTestCase(unittest.TestCase):
+    def test__main__(self, interface_mock):
```

Comment:
I was hoping for you do something like 

     luigi --module mario_jobs.false_positive FalsePositiveLauncher --site=mail.ru --local-scheduler

Like you said wasn't possible before.